Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo

```diff
@@ -1,5 +1,5 @@
-within Buildings.Air.Systems.SingleZone.VAV;
-model ChillerDXHeatingEconomizerController
+within Buildings.Air.Systems.SingleZone.VAV.BaseClasses;
```

Comment:
This rename of the model also needs to be added to the conversion script and the release notes.

Reply:
Added to release notes in https://github.com/lbl-srg/modelica-buildings/pull/2266/commits/f7818c64c2bf782d4910ec5cf6c970a4ee679515 and conversion script in https://github.com/lbl-srg/modelica-buildings/pull/2266/commits/857a3995d9e24e2ec263c493561e5fbd8b0d0307.